src/Scrabble/Search.hs

```diff
@@ -54,6 +56,22 @@ containsAll :: String -> Search
 containsAll s1 s2 = fst $ foldl' f (True, ups s2) (ups s1) where
   f (b,s) c = if elem c s then (b, delete c s) else (False,s)
 
+-- | Like the above, but if it encounters a character in in s1 not in
+--   s2, it deletes a blank and tries again before returning false
+containsAllWithBlanks :: String -> Search
+containsAllWithBlanks s1 s2 = fst $ foldl' f (True, ups s2) (ups s1) where
+    f (b,s) c = if elem c s then (b, delete c s)
+                else if elem '_' s then (b, delete '_' s)
+                else (False, s)
+
+
+countBlanksUsed :: String -> String -> Int
```

Comment:
This isn't a useful search function, and should definitely be moved to `Move`. I think just moving them to `Move` will be easiest. I can deal with it later if I want perfection.